evm/package.json

```diff
@@ -4,15 +4,44 @@
   "license": "MIT",
   "scripts": {
     "build": "truffle build",
-    "test": "truffle test",
-    "lint": "echo \"No linter configured for evm project\" &&  exit 0"
+    "depcheck": "echo 'chainlink' && depcheck --ignore-dirs=build/contracts || true",
```

Comment:
We should ignore all sub-packages here too, like `box` and `v0.5` since they define their own dependencies.